indy_node/test/replay/test_successive_batch_no_state_change.py

```diff
@@ -229,15 +230,17 @@ def patched_com(self, stateRoot):
 
     waitNodeDataEquality(looper, nodeSet[0], *nodeSet[1:])
 
-    keys = {}
-    for _ in range(3):
-        idy, _ = new_identity()
-        keys[idy.identifier] = idy.verkey
-        submit_id_req(idy)
-        looper.runFor(.01)
-
-    # Correct number of uncommitted entries
-    looper.run(eventually(check_uncommitted, 3, retryWait=1))
+    stashers = [n.nodeIbStasher for n in nodeSet]
+    with delay_rules(stashers, icDelay()):
```

Comment:
Why do we prevent View Change only here? Don't we need to prevent the view change for the whole test?

Reply:
@ashcherbakov in fact View Change was offending only this part (or at least it seemed so), so I tried to make minimal change in test behaviour